docs/user/openstack/install_upi.md

```diff
@@ -336,22 +353,29 @@ values:
 
 <!--- e2e-openstack-upi: INCLUDE START --->
 ```sh
-$ python -c 'import yaml
+$ python -c 'import os
+import yaml
+os_net_id = os.getenv("OS_NET_ID")
+path = "common.yaml"
+facts = yaml.safe_load(open(path))[0]["tasks"][0]["set_fact"]
+os_network = facts["os_network"].replace("{{ os_net_id }}", os_net_id)
+os_subnet = facts["os_subnet"].replace("{{ os_net_id }}", os_net_id)
+os_subnet6 = facts["os_subnet6"].replace("{{ os_net_id }}", os_net_id)
```

Comment:
this needs to be defined only when dualstack is enabled, after line 371

Reply:
ack.